bundles/af-core/src/test/resources/form/datepicker/exporter-datepicker.json

```diff
@@ -7,15 +7,13 @@
   "description": "dummy",
   "tooltip": "test-short-description",
   "type": "string",
-  "required": false,
   "enabled": false,
   "constraintMessages": {
     "type": "incorrect type"
   },
   "readOnly": true,
   "default": "2022-02-07",
   "label": {
-    "richText": false,
     "visible": true,
```

Comment:
should `visible` be also removed from here?

Reply:
This is a test collateral, in the test content i have explicitly added the visible property to  verify that if visible is explicitly added using dialog, the generated sling model honors that.

Reply:
resolved as per discussion